spec/acceptance/shared_examples/fixtures_move.rb

```diff
@@ -21,6 +21,7 @@
 
     it 'should recreate empty dirs removed from rsync skeleton by fixtures copy' do
       # TODO: Replace simp-rsync-skeleton install with git clone instead?
+#FIXME remove this when EL6 is removed from simp-rsync-skeleton
       on(master, "mkdir #{fixtures_staging_dir}/assets/rsync_data/rsync/RedHat/6/bind_dns/default/named/var/tmp")
       on(master, "mkdir #{fixtures_staging_dir}/assets/rsync_data/rsync/RedHat/6/bind_dns/default/named/var/log")
     end
```

Comment:
Do these directories actually need to be created any longer?

Reply:
Yes:
	Failure/Error: on(host, "simp environment new --skeleton --no-puppetfile-gen #{opts[:env]}")
     Beaker::Host::CommandFailure:
       Host 'el7' exited with 1 running:
        simp environment new --skeleton --no-puppetfile-gen old_simplib
       Last 10 lines of output were:
       	Applying Puppet permissions (group) to '/var/simp/environments/old_simplib'
       	Applying FACL rules to '/var/simp/environments/old_simplib/rsync'
       	
       	[setfacl --restore=/var/simp/environments/old_simplib/rsync/.rsync.facl] failed with exit status 1:
       	  setfacl: RedHat/6/bind_dns/default/named/var/tmp: No such file or directory
       	  setfacl: RedHat/6/bind_dns/default/named/var/log: No such file or directory
       	
       	ERROR:  Failed to apply FACL rules to /var/simp/environments/old_simplib/rsync